### PR TITLE
Add Fall Old-Fashioned

### PR DESCRIPTION
**Pull request by s-Hale**

### CODE REVIEW

**Review by alfg — CHANGES_REQUESTED** [review]
Hi, can you rename the image to match the recipe?
`fall-old-fashioned.jpg`.

File: src/recipes/fall-old-fashioned.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
+{
+    "name": "Fall Old-Fashioned",
+    "description": "A classic Old-Fashioned tailored to Fall.",
+    "github": "shale",
```
Comment (alfg) [inline src/recipes/fall-old-fashioned.json:4]: Is this the right user?

**Review by alfg — CHANGES_REQUESTED** [review]

File: src/recipes/fall-old-fashioned.json (hunk 1 above)
Comment (s-Hale) [reply src/recipes/fall-old-fashioned.json]: Sorry, needs a hyphen. Fixed.

**Comment by alfg** [comment]
Thanks!

**Pull request merged**